src/TSHelper.ts

```diff
@@ -453,4 +453,28 @@ export class TSHelper {
     public static isDefaultArrayPropertyName(methodName: string): boolean {
         return defaultArrayPropertyNames.has(methodName);
     }
+
+    public static isNonFalsible(type: ts.Type, strictNullChecks: boolean): boolean {
```

Comment:
IsNonFalsible is a pretty confusing name - perhaps it should be reversed to be IsFalsible and checked with a not "!" operator later.

Reply:
Fine by me.